plugins/Readability/data/Call.js

```diff
@@ -0,0 +1,100 @@
+%1
+
+%2
+
+function callReadability(){
+	var loc = document.location;
+	var uri = {
+  		spec: loc.href,
+  		host: loc.host,
+  		prePath: loc.protocol + "//" + loc.host,
+  		scheme: loc.protocol.substr(0, loc.protocol.indexOf(":")),
+  		pathBase: loc.protocol + "//" + loc.host + loc.pathname.substr(0, loc.pathname.lastIndexOf("/") + 1)
+	};
+	var article = new Readability(uri, document).parse();
+	renderPage(article);
+}
+
+function readabilityHead(article){
+	var head = document.createElement('head');
+	document.documentElement.appendChild(head);
+
+	var title = document.createElement('title');
+	title.innerHTML = article.title;
+	head.appendChild(title);
+
+    var css = document.createElement('style');
```

Comment:
Please fix indentation. Somewhere you use tabs and somewhere spaces, I would prefer to use only spaces.